build.py

```diff
@@ -1357,9 +1357,11 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu, target_mach
     if "vllm" in backends:
         # [DLIS-5606] Build Conda environment for vLLM backend
         # Remove Pip install once vLLM backend moves to Conda environment.
+        # [DLIS-5650] Pre-installing torch 2.0.1, since vllm 0.2
+        # requires torch >= 2.0.0, but it doesn't work with torch 2.1.0.
         df += """
 # vLLM needed for vLLM backend
-RUN pip3 install vllm=={}
+RUN pip3 install torch==2.0.1 vllm=={}
 """.format(
```

Comment:
Should we put pinned torch version in the Triton version map?

Reply:
I believe it's by a reason. 
You can confirm it with @oandreeva-nv

Reply:
We actually don't need this `torch==2.0.1` anymore since L81 already has newer version of vLLM, than the one used in 23.10 release. To simplify current process, we can merge it as is and I can follow up and remove this particular commit

Reply:
or, if possible, just commit suggestions

Reply:
I don't see a suggestion...